rust/toolchain.bzl

```diff
@@ -133,6 +151,39 @@ def _get_files(input):
       files += [f for f in i.files]
   return files
 
+def _path_parts(path):
+  """Takes a path and returns a list of its parts with all "." elements removed.
+
+  The main use case of this function is if one of the inputs to _relative()
+  is a relative path, such as "./foo".
+
+  Args:
+    path_parts: A list containing parts of a path.
+
+  Returns:
+    Returns a list containing the path parts with all "." elements removed.
+  """
+  path_parts = path.split("/")
+  return [part for part in path_parts if part != "."]
+
+def relative_path(src_path, dest_path):
```

Comment:
If this function is internal to this bzl file, prefix with `_`

Reply:
It's also used in rust/rust.bzl

Reply:
Ah you're right.